<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinewidagetInStackWidget</class>
 <widget class="QWidget" name="MinewidagetInStackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>1396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="backgroundwidget" native="true">
     <property name="styleSheet">
      <string notr="true">#backgroundwidget
{
	background-color: rgb(255, 255, 255);
}
QPushButton
{
	background-color: rgb(255, 255, 0);
	color:rgb(0,0,0);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_head" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 219, 97);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,4,2,1">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>35</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>35</number>
         </property>
         <item>
          <widget class="QLabel" name="label_HeadIcon">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/public/image/HeadPortait.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_UserName">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>MichaelFeng</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>274</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-radius:20px;
</string>
           </property>
           <property name="text">
            <string>打卡</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>53</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_second" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 219, 97);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWidget" name="widget_First" native="true">
           <layout class="QGridLayout" name="gridLayout" rowstretch="1,2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>40</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(121, 100, 43);
font: 16pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>已连续打卡</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_Second" native="true">
           <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>40</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color: rgb(121, 100, 43);
font: 16pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>记账总天数</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_Third" native="true">
           <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>40</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">color: rgb(121, 100, 43);
font: 16pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>记账总笔数</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_Third_Out" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.994792, y1:0.313, x2:1, y2:1, stop:0 rgba(249, 219, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/MineArea/image/Pic_InMineArea.jpg);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_Four_Out" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 246, 246);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,3,7,1">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/MineArea/image/Pic_Two_InMineArea.jpg);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/MineArea/image/Pic_Three_InMineArea.jpg);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/MineArea/image/Pic_Four_InMineArea.jpg);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="ChangePasswordButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>修改密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ExitLoginButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
              </property>
              <property name="text">
               <string>退出登录</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
